libs/features/jarvis-chat.js

```diff
@@ -278,23 +281,31 @@ const startInitialization = async (config, event) => {
   });
 };
 
-const initJarvisChat = async (config, loadScriptFunction, loadStyleFunction) => {
+const initJarvisChat = async (
+  config,
+  loadScriptFunction,
+  loadStyleFunction,
+  getMetadataFunction,
+) => {
   if (!config?.jarvis) return;
 
   loadScript = loadScriptFunction;
   loadStyle = loadStyleFunction;
+  getMetadata = getMetadataFunction;
+
+  const onDemandMeta = getMetadata('jarvis-on-demand')?.toLowerCase();
+  const onDemand = onDemandMeta ? onDemandMeta === 'on' : config.jarvis.onDemand;
```

Comment:
You can pass "onDemand" directly to `startInitialization`, eliminating the need to evaluate its value again

Reply:
done